### PR TITLE
Vimshell

### PR DESCRIPTION
**Pull request by rocknrollMarc**
Bundle and keymap ,s to open vimshell

### CODE REVIEW

File: vim/settings/vimshell.vim [hunk 1]
```diff
@@ -0,0 +1 @@
+map ,s :sp<CR>:resize 10<CR>:VimShell<CR
```
Comment (skwp) [inline vim/settings/vimshell.vim]: sorry to keep making you change things but this keymap will interfere with other ones starting with `,s` (to see the list, use `:map ,s`

How about we make this `,vs` for vimshell

**Comment by rocknrollMarc** [comment]
sure great no problem I want this to work well :) Im actually a guitarist and a rails dev only thing is Im situated in hamburg germany. Does Reverb have remote devs? would love to work for you guys.

Are you changing it or me ?

**Comment by rocknrollMarc** [comment]
Take a look at nerd-tree git and the webicons bundle Ive got bolth on my vim but what about adding it to yadr?mmakes it look way nicer even in terminal

File: vim/settings/vimshell.vim [hunk 2]
```diff
@@ -0,0 +1 @@
+map ,vs :sp<CR>:resize 10<CR>:VimShell<CR
```
Comment (stevenbarragan) [inline vim/settings/vimshell.vim]: I think you have a typo, you need `>` at the end

**Comment by stevenbarragan** [comment]
We need to install http://github.com/Shougo/vimproc as well, I think it would be nice to install it on the installation process.

File: vim/settings/vimshell.vim (hunk 2 above)
Comment (skwp) [inline vim/settings/vimshell.vim]: agreed looks like a typo

**Comment by skwp** [comment]
sorry to let this sit so long. if it's working please confirm, it definitely looks like you have a typo there in vimshell.vim

**Pull request closed without merging**